pkg/apis/v1beta1/aksnodeclass.go

```diff
@@ -41,8 +49,12 @@ type AKSNodeClassSpec struct {
 	ImageID *string `json:"-"`
 	// ImageFamily is the image family that instances use.
 	// +kubebuilder:default=Ubuntu2204
-	// +kubebuilder:validation:Enum:={Ubuntu2204,AzureLinux}
+	// +kubebuilder:validation:Enum:={Ubuntu,Ubuntu2204,AzureLinux}
```

Comment:
This option I think does need to be put into both v1beta1, v1alpha2, as otherwise I think it can cause problems with the default conversion where users GET w/ v1alpha2 and get an Enum value they weren't expecting. APIServer may then reject modifications to that resource via v1alpha2 because it contains an unsupported enum value for this field.